syntax = "proto3";
package DB.FoundationDB.Proto;

message MetadataTable {
    string sql = 1;
}
